ansible/roles/debops.gitlab_runner/defaults/main.yml

```diff
@@ -126,7 +126,7 @@ gitlab_runner__concurrent: '{{ ansible_processor_vcpus
 # .. envvar:: gitlab_runner__domain [[[
 #
 # The default domain used in different places of the role.
-gitlab_runner__domain: '{{ ansible_domain }}'
+gitlab_runner__domain: '{{ gitlab_domain|d(ansible_domain) }}'
```

Comment:
That will work only if the user has `gitlab_domain` set in the inventory. If you could that this value can be set through the `debops.gitlab` role, it will work only if the `debops.gitlab` role is in the same playbook as `debops.gitlab_runner` role which is not the case in DebOps.

A better solution would be to expose the GitLab domain and FQDN via the Ansible local facts defined by the `debops.gitlab` role, then the `debops.gitlab_runner` role could use them in the above way to configure itself... However since the Runner(s) would probably be configured and used on separate hosts from the main GitLab instance for security and performance, that probably won't work. So instead, perhaps the role could check the DNS SRV resource records, say `_gitlab._tcp.example.org` to find the domain of the GitLab host and configure the Runners that way. Of course this requires setting up that DNS record first, but might be a useful feature.

Reply:
Based on your works for `_imap._tcp` and so on, updated the defaults accordingly.
Please check on your side if it still works ;)